plugins/modules/monitoring/pagerduty_user.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Zainab Alsaffar <Zainab.Alsaffar@mail.rit.edu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: pagerduty_user
+short_description: Manage a user account on PagerDuty
+description:
+    - This module manages the creation/removal of a user account on PagerDuty.
+version_added: '1.3.0'
+author: Zainab Alsaffar (@zanssa)
+requirements:
+    - pdpyras python module = 4.1.1
+    - PagerDuty API Access
+options:
+    access_token:
+        description:
+            - An API access token to authenticate with the PagerDuty REST API.
+        required: true
+        type: str
+    pd_user:
+        description:
+            - Name of the user in PagerDuty.
+        required: true
+        type: str
+    pd_email:
+        description:
+            - The user's email address.
+            - I(pd_email) is the unique identifier used and cannot be updated using this module.
+        required: true
+        type: str
+    pd_role:
+        description:
+            - The user's role.
+        choices: ['global_admin', 'manager', 'responder', 'observer', 'stakeholder', 'limited_skateholder', 'restricted_access']
```

Comment:
I guess this is a typo:
```suggestion
        choices: ['global_admin', 'manager', 'responder', 'observer', 'stakeholder', 'limited_stakeholder', 'restricted_access']
```
(Same further below)

Reply:
huh. It is. fixed now!